core/designsystem/src/androidTest/java/com/google/samples/apps/nowinandroid/core/designsystem/ThemeTest.kt

```diff
@@ -136,7 +147,15 @@ class ThemeTest {
                     DarkDefaultColorScheme
                 }
                 assertColorSchemesEqual(colorScheme, MaterialTheme.colorScheme)
-                val gradientColors = GradientColors()
+                val gradientColors = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+                    GradientColors()
+                } else {
+                    GradientColors(
```

Comment:
Seems like there's quite a bit of repetition of this declaration - refactor into class variable? Or leave as-is, redundancy in tests is fine.

Reply:
Done